### PR TITLE
Add navigation

### PR DESCRIPTION
**Pull request by teifler**
Add overall grid layout and navigation

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  &.active {
+    color: #2f66f6;
+    opacity: 1;
+    svg {
+      fill: #2f66f6;
+    }
+  }
+`;
+const Image = styled.div``;
```
Comment (J-Massmann) [inline src/components/Navbar.js:78]: If you name the constant 'Image' you should use an `img`-Element.

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+  return (
+    <NavContainer>
+      <LinkTo to="/">
```
Comment (J-Massmann) [inline src/components/Navbar.js:8]: You are nesting many Elements inside divs. Not sure if that is really necessary...
As an alternative you could do something like this
`<button>`
   ` <Link>`
     ` `
  ` </Link>`
`</button>`

**Review by J-Massmann — COMMENTED** [review]
All in all it looks good, just some tiny improvements imo 👍

File: src/App.js [hunk 3]
```diff
@@ -47,48 +50,79 @@ function App() {
...
+
 const ErrorMessage = styled.h3`
   color: red;
 `;
 
 const SpinnerLogo = styled.img`
-  margin-top: 40vh;
+  position: absolute;
+  top: 50%;
+  left: 50%;
```
[the remark this answers is not in the document]
Comment (teifler) [reply src/App.js:127]: Thank you I happily added that!

File: README.md [hunk 4]
```diff
@@ -1,3 +1,15 @@
+# CryptoCloud track your favorite CryptoCurrency's Tracker - Build with React, Chart JS
+
+Cryptocloud is a cryptocurrency
```
[the remark this answers is not in the document]
Comment (teifler) [reply README.md]: oops, thats definitly not correct :D

**Review by ttaannjjaa — APPROVED** [review]
looks very good to me, some little comments we talked about

**Pull request merged**